config/latest-mt-debug-howso.yml

```diff
@@ -4,6 +4,6 @@ Howso:
      debug: true
      check_version: false 
      amalgam:
-        library_postfix: -mt
+        library_postfix: -st
         trace: true 
-        execution_trace_dir: ./traces/
\ No newline at end of file
+        execution_trace_dir: ./traces/ 
```

Comment:
Is this an extra space at the end? Probably doesn't matter but probably good to clean it up.